src/components/forms/Input.js

```diff
@@ -58,8 +62,8 @@ export default function Input({
   }
 
   return (
-    <div className="col-span-full">
-      <label htmlFor={id} className="block text-sm font-medium leading-6">
+    <div className={"col-span-full " + className}>
```

Comment:
please use `classNames` for this also

Reply:
I have integrate the usage of classnames.

Reply:
Thank you